docker_test.go

```diff
@@ -1019,6 +1137,11 @@ func sendWHEvents(wdt *wht.WareHouseDestinationTest) {
 	// Sending identify event
 	if identify, exists := wdt.EventsCountMap["identifies"]; exists {
 		for i := 0; i < identify; i++ {
+			messageId := wdt.MessageId
```

Comment:
Move this to wdt.GetMessageId(). Because same logic.

Reply:
Created new function